packages/fluentui/react-northstar/src/components/Form/FormCheckbox.tsx

```diff
@@ -0,0 +1,46 @@
+import { compose, ComponentWithAs, ShorthandConfig } from '@fluentui/react-bindings';
+import { commonPropTypes } from '../../utils';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import FormField, { FormFieldProps, FormFieldStylesProps } from './FormField';
+import Checkbox, { CheckboxProps } from '../Checkbox/Checkbox';
+import { ShorthandValue } from 'src/types';
+import { TextProps } from '../Text/Text';
+
+interface FormCheckboxOwnProps extends Omit<CheckboxProps, 'styles' | 'accessibility'> {
+  checkboxLabel?: ShorthandValue<TextProps>;
+}
+
+export interface FormCheckboxProps extends FormFieldProps, FormCheckboxOwnProps {}
+export type FormCheckboxStylesProps = never;
+
+export const FormCheckboxClassName = 'ui-form-checkbox';
+
+const FormCheckbox = compose<'div', FormCheckboxProps, FormCheckboxStylesProps, FormFieldProps, FormFieldStylesProps>(
+  FormField,
+  {
+    className: FormCheckboxClassName,
+    displayName: 'FormCheckbox',
+    overrideStyles: true,
+    shorthandConfig: {
+      mappedProp: 'label',
+    },
+    handledProps: ['label'],
+    slotProps: ({ checkboxLabel }) => ({
+      control: {
+        as: Checkbox,
+        label: checkboxLabel,
```

Comment:
@layershifter @mnajdova I had to create this `checkboxLabel` prop because I was expecting that when adding `label` to `handledProps` here it wouldn't be passed to `FormField`, but it was and then it was rendering label twice ( once for the `FormField` label slot and once for the `Checkbox` label slot. Any better way to solve this?

Reply:
Adding it only as handledProp won’t help because in the FormField it is used in the create shorthand.. If the FormCheckbox should not render the label you can probably force null rendering by adding some simple empty rendering component in the slots option

Reply:
I tried something like:

```typescript
slotProps: ({ checkboxLabel }) => ({
      label: null,
      control: {
        as: Checkbox,
        label: checkboxLabel,
      },
    }),
```

But didn't quite worked

Reply:
Solved